src/main/scala/subsystem/HasTiles.scala

```diff
@@ -175,6 +175,7 @@ trait CanAttachTile {
     connectOutputNotifications(domain, context)
     connectInputConstants(domain, context)
     connectTrace(domain, context)
+    connectTraceSinkDMA(domain, context)
```

Comment:
Add a comment here to explain the difference between `connectTrace` and `connectTraceSinkDMA`